app/styles/_tables.less

```diff
@@ -151,8 +151,8 @@
   th#severity {width:10px;padding:0;}
   th#reason {width:150px;}
   th#message   {width:100%;} // ensures it gets remaining width
-  th#kind-name,th#message  {
-    .word-break;
```

Comment:
No we switched it not to do that a while back. There is a `word-break-all` that behaves differently.